src/components/Loading.react.js

```diff
@@ -53,7 +55,13 @@ export default class Loading extends Component {
         const Spinner = isLoading && getSpinner(spinnerType);
 
         return (
-            <div style={isLoading ? hiddenContainer : {}}>
+            <div
+                className={isLoading ? null : className}
```

Comment:
In the original behavior, modified in #740 which introduced the regression, the `className` was either applied to 
- the Spinner, if in a loading state
- the `this.props.children` div parent, otherwise (at least, if `this.props.children` wasn't an `Object` or `Function`, for some reason. If it was, no `className` would be applied. See below for these three conditions.)

I'm attempting to mimic this behaviour here. Ideally this would be two different props, in the pattern of `style` and the new (in this PR) `parent_style`, but in order to fix the regression and have current apps working without breaking changes I think we need to replicate the old behaviour here.

This was the previous render return value:

https://github.com/plotly/dash-core-components/blob/aa4fce7b739de4c91fbae9e746c0b06883dd8569/src/components/Loading.react.js#L40-L73